app/code/Magento/Customer/Controller/Address.php

```diff
@@ -89,12 +95,14 @@ public function __construct(
         \Magento\Framework\Reflection\DataObjectProcessor $dataProcessor,
         \Magento\Framework\Api\DataObjectHelper $dataObjectHelper,
         \Magento\Framework\Controller\Result\ForwardFactory $resultForwardFactory,
-        \Magento\Framework\View\Result\PageFactory $resultPageFactory
+        \Magento\Framework\View\Result\PageFactory $resultPageFactory,
+        ?\Magento\Customer\Api\CustomerRepositoryInterface $customerRepository = null
     ) {
         $this->_customerSession = $customerSession;
         $this->_formKeyValidator = $formKeyValidator;
         $this->_formFactory = $formFactory;
         $this->_addressRepository = $addressRepository;
+        $this->customerRepository= $customerRepository;
```

Comment:
Please add a space before the "=" and get an instance of customer reporsitory via object manager
```suggestion
        $this->customerRepository= $customerRepository ?: \Magento\Framework\App\ObjectManager::getInstance()->get(\Magento\Customer\Api\CustomerRepositoryInterface::class);;
```

Reply:
isn't it too dirty? why my solution with `di.xml` doesnt fit?

Reply:
See https://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/#php for details

Reply:
ok, i see.. sorry

Reply:
Thank you!